app/views/service_providers/_service_provider_table.erb

```diff
@@ -0,0 +1,16 @@
+<% @service_providers.each do |type, apps| %>
+  <h1 class="usa-display"><%= type %></h1>
+
+  <% unless apps.blank? %>
+  <table class="usa-table">
```

Comment:
[style nitpick] indent the `table` block to show nesting

Reply:
Adding the indent.